packages/vscode-extension/src/loggers/VSConsoleLogger.ts

```diff
@@ -41,7 +41,7 @@ export default class VSConsoleLogger extends ConsoleLogger {
 	// Output from VSCode doesn't accept colors, for the moment we would pring in default white
 	// We could explore some workarounds in future like creating a Terminal, importing a new library or just implment it ourselves
 	println(message: string): void {
-		if (this.addExecutionDetailsToLog) {
+		if (this.addExecutionDetailsToLog && message !== '') {
```

Comment:
yes. what i meant is that we could avoid calling println() when the message is null or empty to begin with. Or at least we could just add a check in the start of the method:
if (message == '' or null )
{
    return;
}

what do you think?

Reply:
I guess there is no point to process empty string. i think with the way it is now, it prints an empty line to console which I don't think we want.

Reply:
I tried going with that but the problem is that the output from different actions will not have a blank line between them.

```
timestamp - acp01
blah blah message
timestamp - suiteapp01
blah blah message
```

So it'll be hard to distinguish information from different action.

Reply:
so we are adding blank lines ourselves by calling println('')?
If that is the case, cannot we just add the blank line when adding the timestamp inside println:
**output.appendLine(message);**
output.appendLine(this.getExecutionDetails());

this way it will always have a blank line before the timestamp.

Reply:
Won't there be a message logged before the execution details?

Also, I was trying to touch as little of existing code as possible.